### PR TITLE
Review for Product-Landing-Page project

### PR DESCRIPTION
**Pull request by ZohalMirkhel**
**Description**
This pull request contains the webpage I have created for our fifth project, after checking it in freeCodeCamp.

**Features**
Header and Navigation: Logo and brand name "Afghan Elegance"
Navigation links: Trending, History, Services, Contact, User and shopping cart icons

**Banner Section:**
Main title: "AFGHANI CLOTHES"
Subtitle: "Experience the timeless elegance of Afghan craftsmanship with our exquisite collection of traditional clothing and accessories."
Email subscription form
Search Bar: Search input and button for product search
Trending Section: Product grid showcasing six items with images, prices, descriptions, and "BUY" buttons

**History Section:**
1. Text describing the rich history of Afghan clothing
2. Embedded YouTube video

**Services Section:**
Four service descriptions: Fast Shipping, Easy Returns, 24/7 Support, Affordable Prices

**Footer Section:**
Contact information: Email, phone, address

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+              <li><a class="nav-link" href="#trending">Trending</a></li>
+              <li><a class="nav-link" href="#history">Hoistory</a></li>
+              <li><a class="nav-link" href="#services">Services</a></li>
+              <li><a class="nav-link" href="#footer">Contact</a></li>
+            </ul>
+            <div class="menu-icons">
+              <i class="fas fa-user-circle"></i>
+              <i class="fa fa-shopping-cart"></i>
+            </div>
+          </nav>
```
Comment (ahzamir) [inline index.html:39]: - Your #nav-bar should always be at the top of the viewport.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @ZohalMirkhel 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Wow you have done hard work and I can see the passion and interest in your code and design although your code and your project have passed lots of the requirements but still miss a few requirements in order to get the final approval.

- Copy your HTML file code and paste it to the freecodecamp website Product Landing Page project to see which requirements your project is missing solve those missing requirements and mention me here using @ahzamir to re-review your project.

- Check the Comments below for more details.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
- 
image

✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
❌ You have followed and completed the Product Landing Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your project has a professional README.md file
✔️Your code is clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by ZohalMirkhel** [comment]
@ahzamir Sir, I worked on the project as you suggested; the code passed freecodecamp's check this time as well. Kindly take a look.

**Review by ahzamir — APPROVED** [review]
### Hi @ZohalMirkhel  🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍

image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Product Landing Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**